### PR TITLE
Remove code that is used for buildBuildImages

### PR DESCRIPTION
**Pull request by william-msft**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: images/build/Dockerfiles/ltsVersions.buster.Dockerfile [hunk 1]
```diff
@@ -86,17 +86,6 @@ RUN set -ex \
...
-    && . $buildDir/__golangVersions.sh \
-    && $imagesDir/installPlatform.sh golang $GOLANG16_VERSION \
-    && ln -sfn $GOLANG16_VERSION 1.16 \
-    && $imagesDir/installPlatform.sh golang $GOLANG17_VERSION \
-    && cd /opt/golang \
-    && . $buildDir/__golangVersions.sh \
-    && ln -sfn $GOLANG17_VERSION lts \
-    # Install .NET Core SDKs
-    && nugetPackagesDir="/var/nuget" \
-    && mkdir -p $nugetPackagesDir \
```
Comment (arroyc) [inline images/build/Dockerfiles/ltsVersions.buster.Dockerfile:99]: why removing these lines?

File: images/__common.sh [hunk 2]
```diff
@@ -24,10 +24,9 @@ function downloadFileAndVerifyChecksum() {
     rm -f $headersFile
     echo
     echo "Verifying checksum..."
-
     checksumcode="sha512sum"
     if [ "$platformName" == "golang" ];then
         checksumcode="sha256sum"
-   fi
+    fi
```
Comment (arroyc) [inline images/__common.sh:30]: nit: remove space

Comment (william-msft) [reply images/__common.sh:30]: @arroyc if I remove the space, the `fi` wont alight with the `if`

File: images/build/Dockerfiles/ltsVersions.buster.Dockerfile (hunk 1 above)
Comment (william-msft) [reply images/build/Dockerfiles/ltsVersions.buster.Dockerfile:99]: Updated, thanks

**Review by arroyc — APPROVED** [review]

**Pull request merged**